gradle/libs.versions.toml

```diff
@@ -91,6 +91,10 @@ node-plugin = "7.0.1"
 commons-cli = "1.2"
 sun-activation-version = "1.2.0"
 error-prone = "3.1.0"
+woodstox-core = "5.3.0"
+mail = "1.4.1"
+rome = "1.0"
+jettison = "1.1"
```

Comment:
If we need to add these new dependencies, we should also update the license.bin file.

Reply:
added